sample-app/src/main/java/com/sinch/sample/webhooks/verification/VerificationController.java

```diff
@@ -39,38 +40,36 @@ public String VerificationEvent(
 
     // ensure valid authentication to handle request
     var validAuth =
-        sinchClient
-            .verification()
-            .webhooks()
-            .validateAuthenticationHeader(
-                // The HTTP verb this controller is managing
-                "POST",
-                // The URI this controller is managing
-                "/VerificationEvent",
-                // request headers
-                headers,
-                // request payload body
-                body);
+        webhooks.validateAuthenticationHeader(
```

Comment:
Very nice to promote the user of services directly. It fits perfectly with the ADR 🙌